frontend/src/app/modules/authentication/logo/logo.component.sass

```diff
@@ -19,11 +19,10 @@
     align-items: center
 
 .logo-img
-    height: 118px
-    width: 118px
+    width: 281.34px
+    height: 192.82px
```

Comment:
Is it possible to avoid fractional pixels becouse different browsers can round them differently?